### PR TITLE
[CARBONDATA-1770] Updated documentaion for data-management-on-carbondata.md and useful-tips-on-carbondata.md

### PR DESCRIPTION
**Pull request by vandana7**
While reviewing PR #1534, still there exist some changes which needs to be fixed. I have done those fixes in this PR, please review.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/1386/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/1833/

File: docs/data-management-on-carbondata.md [hunk 1]
```diff
@@ -294,11 +294,11 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
     ```
     NOTE: ALL_DICTIONARY_PATH and COLUMNDICT can't be used together.
     
-  - **DATEFORMAT:** Date format for specified column.
+  - **DATEFORMAT/TIMESTAMPFORMAT:** Date and Timestamp format for specified column.
 
     ```
-    OPTIONS('DATEFORMAT'='column1:dateFormat1, column2:dateFormat2')
-    ```
+    OPTIONS('dateformat' = 'yyyy-MM-dd','timestampformat'='yyyy/MM/dd HH:mm:ss')
```
Comment (chenliang613) [inline docs/data-management-on-carbondata.md]: please use uppercase for "dateformat,timestampformat".

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/1388/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/1390/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/1837/

**Comment by chenliang613** [comment]
LGTM

**Pull request closed without merging**